packages/camera/camera/pubspec.yaml

```diff
@@ -4,7 +4,7 @@ description: A Flutter plugin for controlling the camera. Supports previewing
   Dart.
 repository: https://github.com/flutter/plugins/tree/master/packages/camera/camera
 issue_tracker: https://github.com/flutter/flutter/issues?q=is%3Aissue+is%3Aopen+label%3A%22p%3A+camera%22
-version: 0.9.4+5
+version: 0.9.4+6
```

Comment:
Note to myself: should not bump the version in this PR. bump it in next PR.

Reply:
@jmagman I removed this one but it's giving me error saying changelog and yaml versions does not match. Maybe i should not update the changelog in this PR?

Reply:
synced with @cyanglaz offline that we can use `## NEXT` in the changelog